db/schema_tables.cc

```diff
@@ -254,6 +255,32 @@ schema_ptr keyspaces() {
     return schema;
 }
 
+schema_ptr scylla_keyspaces() {
+    static thread_local auto schema = [] {
+        schema_builder builder(generate_legacy_id(NAME, SCYLLA_KEYSPACES), NAME, SCYLLA_KEYSPACES,
+        // partition key
+        {{"keyspace_name", utf8_type}},
+        // clustering key
+        {},
+        // regular columns
+        {
+            {"storage", map_type_impl::get_instance(utf8_type, utf8_type, false)},
+        },
+        // static columns
+        {},
+        // regular column name type
+        utf8_type,
+        // comment
+        "scylla-specific keyspace definitions"
```

Comment:
This phrasing is a bit ambigous - are those "scylla-specific-keyspace definitions" or "scylla-specific keyspace-definitions"? :-) In other words, do these define Scylla-specific keyspaces, *or*, add Scylla-specific features to keyspaces which already should be listed in the regular "keyspaces" table?

I think the answer is the latter - the same keyspace will need to be listed in both "keyspaces" and "scylla_keyspaces".